Makefile

```diff
@@ -122,4 +122,4 @@ coverage: ## show the coverage report
 
 .PHONY: clean
 clean: ## clean up the environment by deleting the .venv, dist, eggs, mypy caches, coverage info, etc
-	rm -rf .venv $(DEPS) dist *.egg-info .mypy_cache build .pytest_cache .coverage runinfo_* $(WORKQUEUE_INSTALL)
+	rm -rf .venv $(DEPS) dist *.egg-info .mypy_cache build .pytest_cache .coverage runinfo $(WORKQUEUE_INSTALL)
```

Comment:
i think this is a general tidyup. if so, it would be a good separate PR

Reply:
You mean `runinfo_*` is a general tidyup? But when I test it locally, the dir created by `make test` is `runinfo` instead of `runinfo_*`

Reply:
I mean that this change to the `rm` commandline is a general fix you could submit as a separate PR.

Reply:
runinfo_* directories were where test logs used to get stored a long time ago but that kind of directory hasn't existed for a long time.

Reply:
Done in: https://github.com/Parsl/parsl/pull/3236